### PR TITLE
refactor(ui): add real links to alerting documentation

### LINKED ISSUE #14765: Add links to documentation for alerting
Help button on Check EO. 
documentation links on empty check/rule/endpoint columns.

### PR DESCRIPTION
**Pull request by alexpaxton**
Closes #14765 

Screen Shot 2019-09-09 at 5 08 07 PM
Screen Shot 2019-09-09 at 5 09 06 PM

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/alerting/components/builder/HelpButton.tsx [hunk 1]
```diff
@@ -5,7 +5,12 @@ import React, {FunctionComponent} from 'react'
 import {Button} from '@influxdata/clockface'
 
 const HelpButton: FunctionComponent = () => {
-  const handleClick = () => {}
+  const handleClick = () => {
+    const newTab = window.open(
+      'https://github.com/influxdata/docs-v2/blob/monitor-alert/content/v2.0/monitor-alert/checks/create.md#configure-the-check'
+    )
```
Comment (noramullen1) [inline ui/src/alerting/components/builder/HelpButton.tsx]: That's just the markdown github link--live link will be https://v2.docs.influxdata.com/v2.0/monitor-alert/checks/create/#configure-the-check

File: ui/src/alerting/components/builder/HelpButton.tsx [hunk 2]
```diff
@@ -5,7 +5,12 @@ import React, {FunctionComponent} from 'react'
 import {Button} from '@influxdata/clockface'
 
 const HelpButton: FunctionComponent = () => {
-  const handleClick = () => {}
+  const handleClick = () => {
+    const newTab = window.open(
+      'https://github.com/influxdata/docs-v2/blob/monitor-alert/content/v2.0/monitor-alert/checks/create.md#configure-the-check'
```
Comment (timhallinflux) [inline ui/src/alerting/components/builder/HelpButton.tsx]: not the right URL

File: ui/src/alerting/components/builder/CheckMessageCard.tsx [hunk 3]
```diff
@@ -74,6 +74,15 @@ const CheckMessageCard: FC<Props> = ({check, onUpdateTimeMachineCheck}) => {
             <code>{'${string(v: r.numericColumn)}'}</code> Functions can be used{' '}
             as well
           </p>
+          <p>
+            Need help? Check out the Status Message Template{' '}
+            <a
+              href="https://github.com/influxdata/docs-v2/blob/monitor-alert/content/v2.0/monitor-alert/checks/create.md#flux-only-interpolates-string-values"
```
Comment (timhallinflux) [inline ui/src/alerting/components/builder/CheckMessageCard.tsx]: not the right URL

Comment (noramullen1) [inline ui/src/alerting/components/builder/CheckMessageCard.tsx]: Link should be https://v2.docs.influxdata.com/v2.0/monitor-alert/checks/create/#flux-only-interpolates-string-values

**Comment by alexpaxton** [comment]
@timhallinflux just updated with the correct links from @noramullen1

**Pull request merged**